fiftyone/utils/sapiens.py

```
@@ -0,0 +1,280 @@
+        model = foz.load_zoo_model("sapiens2-1b-pose-torch")
+        dataset.apply_model(
+            model, label_field="pose", prompt_field="persons"
+        )
+
+        session = fo.launch_app(dataset)
+
+    Args:
+        config: a :class:`Sapiens2PoseModelConfig`
+    """
+
+    def __init__(self, config):
```

Ruff (0.15.13)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)